interactive_ai/services/director/app/storage/mappers/project_configuration_mapper.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (C) 2022-2025 Intel Corporation
+# LIMITED EDGE SOFTWARE DISTRIBUTION LICENSE
+
+from geti_configuration_tools.project_configuration import (
+    AutoTrainingParameters,
+    ProjectConfiguration,
+    TaskConfig,
+    TrainingParameters,
+)
+
+from geti_types import ID
+from iai_core.repos.mappers.mongodb_mapper_interface import IMapperSimple
+from iai_core.repos.mappers.mongodb_mappers.id_mapper import IDToMongo
+
+
+class TaskConfigToMongo(IMapperSimple[TaskConfig, dict]):
+    """MongoDB mapper for `TaskConfig` entities"""
+
+    @staticmethod
+    def forward(instance: TaskConfig) -> dict:
+        return {
+            "task_id": IDToMongo.forward(ID(instance.task_id)),
+            "training": instance.training.model_dump_json(),
+            "auto_training": instance.auto_training.model_dump_json(),
```

Comment:
I'm wondering if we can write the mappers in a way that is agnostic to the specific field names defined in the config, so that we don't have to update the mapper everytime we add/remove a configurable parameter. Maybe `dict(instance)` works out of the box, since it's a `BaseModel`?